docs/_posts/2017-10-19-diversity-open-source.markdown

```diff
@@ -0,0 +1,44 @@
+
+So yeah, there's a bunch of big problems plaguing open source. Fortunately, there's a couple of things we can do to make it easier and more comfortable for people, especially minorities, that have never before contributed to any open source project before, to contribute to our projects.
+
+## Add a Code of Conduct, and enforce it
+
+This might seem like one of the easiest steps to do, but it actually requires a lot of dedication to pull through with. Basically, a Code of Conduct is a document detailing what is and what isn't acceptable behavior in your project. A good Code of Conduct also details enforcement procedures, that means how the person violating the Code of Conduct gets dealt with. This is the point at which I've seen a looooot of projects fail. It's easy enough to copy-paste a Code of Conduct into your project, but it's just, if not more important to be clear on how to enforce it. Inconsistent, or worse, nonexistent enforcement is just going to scare off newcomers even more!
+
+The most widely adopted Code of Conduct is the [Contributor Covenant](https://www.contributor-covenant.org/). It's a very good catch-all document, but it is a bit light in the enforcement section, so I'd recommend to flesh it out by yourself, be it by means of adding contact information or expanding the enforcement rules.
+
+No matter which Code of Conduct you pick, the most important thing is to actually _read it for yourself_. The worst thing in open source is a maintainer that doesn't know when they've violated their own Code of Conduct.
+
+## Document your contributing workflow
+
+The problem that puts people off the most is incomplete or missing documentation, as revealed through GitHub's [open source survey](http://opensourcesurvey.org/2017). A very popular myth in programming is that good code explains itself, which might be true, but only for the person writing it. It's important, especially once you put your project out there for the world to see, to document not only your code, but also the process by which you maintain it. Otherwise, it's going to be extremely hard for newcomers to even figure out where to begin contributing to your project.
+
+Jekyll has [an entire section of its docs](/docs/contributing) dedicated to information on how to contribute for this very reason. Every documentation page has a link to directly edit and improve it on GitHub. It's also important to realize that not all contributions are code. It can be documentation, it can be reviewing pull requests, but it can also just be weighing into issues, and all of this should be recognized in the same way.
```

Comment:
It would be interesting to look up how many PR's merged in the last year were documentation. I bet well over half of PR's to this project have been documentation-related! Very important form of contribution. ✨

Reply:
github search says 204!

<img width="478" alt="image" src="https://user-images.githubusercontent.com/6078550/31811308-65ae4d76-b57f-11e7-86b9-256e8757588d.png">

Reply:
so yeah, more than half of all pull requests